app/src/features/Batches/BatchList.tsx

```diff
@@ -0,0 +1,160 @@
+import React, { useState } from "react";
+
+import {
+  Accordion,
+  AccordionDetails,
+  AccordionSummary,
+  Button,
+  CircularProgress,
+  Typography,
+} from "@material-ui/core";
+import { makeStyles } from "@material-ui/core/styles";
+import { CancelOutlined, ExpandMore, Replay } from "@material-ui/icons";
+import Pagination from "@material-ui/lab/Pagination";
+import { useTranslation } from "react-i18next";
+
+import { useApiBatchesListQuery } from "services/api/endpoints";
+
+import BatchCancel from "./BatchCancel";
+import BatchErrors from "./BatchErrors";
+
+const useStyles = makeStyles((theme) => ({
+  accordionSummaryContent: {
+    width: "100%",
+    display: "flex",
+    justifyContent: "space-between",
+    alignItems: "center",
+  },
+  accordionSummary: {
+    margin: 0,
+    minHeight: 56,
+    "& > .MuiAccordionSummary-content": {
+      minHeight: 56,
+      margin: 0,
+      transition: "none",
+      "&.Mui-expanded": {
+        minHeight: 56,
+      },
+    },
+    "&.MuiAccordionSummary-root": {
+      borderBottom: `1px solid ${theme.palette.background.default}`,
+      minHeight: 56,
+      margin: 0,
+      transition: "none",
+    },
+  },
+  button: {
+    textTransform: "none",
+    marginLeft: theme.spacing(1),
+  },
+  accordionDetails: {
+    display: "flex",
+    flexDirection: "column",
+    marginTop: theme.spacing(0.5),
+    marginBottom: theme.spacing(1),
+  },
+  accordion: {
+    "&.Mui-expanded": {
+      margin: 0,
+      marginBottom: theme.spacing(0.5),
+    },
+    "&:before": {
+      display: "none",
+    },
+    borderRadius: theme.shape.borderRadius,
+    marginBottom: theme.spacing(0.5),
+  },
+  pagination: {
+    margin: `${theme.spacing(1)}px 0px`,
```

Comment:
Not sure again, but this should be supported by `theme.spacing` method

```suggestion
    margin: theme.spacing(1, 0),
```